vendor/sigs.k8s.io/cluster-api/pkg/controller/machine/controller.go

```diff
@@ -128,11 +142,39 @@ func (c *MachineControllerImpl) Reconcile(machine *clusterv1.Machine) error {
 	}
 	if exist {
 		glog.Infof("reconciling machine object %v triggers idempotent update.", name)
-		return c.update(m)
+		err = c.update(m)
+		if err != nil {
+			requeueErr, ok := err.(*RequeueAfterError)
+			if ok {
+				glog.Infof("actuator returned requeue after error: %v", requeueErr)
```

Comment:
s/requeue after/requeue-after/ so that it reads a little more clearly that the error returned indicated that there should be a requeue after a period of time as opposed to a requeue after an error.